server/api/routes/userMeeting.cjs

```diff
@@ -1,33 +1,107 @@
 const router = require('express').Router({ mergeParams: true });
+const { Meeting } = require('../../db/index.cjs');
+const { requireToken, sameUserOrAdmin } = require('../authMiddleware.cjs');
 
-router.get('/', (req, res, next) => {
+router.get('/', requireToken, sameUserOrAdmin, async (req, res, next) => {
   /**
    * GET /api/user/:userId/meeting
    * return a list of all meetings associated with userId
    */
-
-  res.send(`hello from /api/user/${req.params.userId}/meeting`);
+  try {
+    const userMeetings = await Meeting.findAll({
```

Comment:
(just taking the note from your own catch - well done) -- need to check userId for existence (User.findByPk(:userId))